lib/spack/spack/test/ci.py

```diff
@@ -476,16 +476,30 @@ def _fail(self, args):
 
 
 def test_ci_create_buildcache(tmpdir, working_env, config, mock_packages, monkeypatch):
```

Comment:
This test is testing just a particular implementation, not an invariant or property. That makes it a completely useless test, cause a refactor breaks it.